src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -416,3 +418,21 @@ export function getTileName(x: number, y: number, gridSize: GridSize): string {
   const tileId = `${`${tileY}`.padStart(nbDigits, '0')}${`${tileX}`.padStart(nbDigits, '0')}`;
   return `${sheetCode}_${gridSize}_${tileId}`;
 }
+
+/**
+ * Validate if a TIFF contains only 8 bits bands.
+ *
+ * @param tiff
+ */
+export async function validate8BitsTiff(tiff: Tiff): Promise<void> {
+  const baseImage = tiff.images[0];
+  if (baseImage === undefined) throw new Error(`Can't get base image for ${tiff.source.url}`);
+
+  const bitsPerSample = await baseImage.fetch(TiffTag.BitsPerSample);
+  if (bitsPerSample == null) {
+    throw new Error(`Failed to extract band information from ${tiff.source.url}`);
+  }
+
+  if (!bitsPerSample.every((currentNumberBits) => currentNumberBits === 8))
+    throw new Error(`${tiff.source.url} is not a 8 bits TIFF`);
```

Comment:
if things are multi lined its generally a good idea to include brackets.

```suggestion
  if (!bitsPerSample.every((currentNumberBits) => currentNumberBits === 8)) {
    throw new Error(`${tiff.source.url} is not a 8 bits TIFF`);
  }
```

Reply:
This seems like the sort of things our linter should enforce.

Reply:
It should be covered by lint I'm not sure why its not enforcing brackets here.